src/core/tokenHandler.js

```diff
@@ -106,64 +106,57 @@ export default function tokenHandlerFactory(options) {
          * @param {String} newToken
          * @returns {Promise<Boolean>} - resolves true if successful
          */
-        setToken: function setToken(newToken) {
+        setToken(newToken) {
             return tokenStore.enqueue(newToken);
         },
 
         /**
          * Extracts tokens from the Client Config which should be received on every page load
          * @returns {Promise<Boolean>} - resolves true when completed
          */
-        getClientConfigTokens: function getClientConfigTokens() {
-            var self = this;
-            var clientTokens = _.map(module.config().tokens, function(serverToken) {
-                return {
-                    value: serverToken,
-                    receivedAt: Date.now()
-                };
-            });
+        getClientConfigTokens() {
+            const {tokens} = module.config();
+            const clientTokens = (tokens || []).map(serverToken => ({
+                value: serverToken,
+                receivedAt: Date.now()
+            }));
 
             // Record that this function ran:
             clientConfigFetched = true;
 
-            return Promise.resolve(clientTokens).then(function(newTokens) {
-                // Add the fetched tokens to the store
-                // Uses a promiseQueue to ensure synchronous adding
-                var setTokenQueue = promiseQueue();
+            return Promise.resolve(clientTokens)
+                .then(newTokens => {
+                    // Add the fetched tokens to the store
+                    // Uses a promiseQueue to ensure synchronous adding
+                    const setTokenQueue = promiseQueue();
 
-                _.forEach(newTokens, function(token) {
-                    setTokenQueue.serie(function() {
-                        return self.setToken(token);
-                    });
-                });
+                    newTokens.forEach(token => setTokenQueue.serie(() => this.setToken(token)));
 
-                return setTokenQueue.serie(function() {
-                    return true;
+                    return setTokenQueue.serie(() => true);
                 });
-            });
         },
 
         /**
          * Clears the token store
          * @returns {Promise<Boolean>} - resolves to true when cleared
          */
-        clearStore: function clearStore() {
+        clearStore() {
             return tokenStore.clear();
         },
 
         /**
          * Getter for the current queue length
          * @returns {Promise<Integer>}
          */
-        getQueueLength: function getQueueLength() {
+        getQueueLength() {
             return tokenStore.getSize();
         },
 
         /**
          * Setter for maximum pool size
          * @param {Integer} size
```

Comment:
Not a big deal, but can we do this?

Reply:
well, did not checke the existing annotations :)